pallets/bridge-mapping/src/tests.rs

```diff
@@ -4,65 +4,69 @@ use frame_support::{assert_ok, assert_err};
 #[test]
 fn set_resource_adds_to_storage() {
     new_test_ext().execute_with(|| {
-        let admin       = Origin::root();
-        let resource_id = 1;
-        let local_addr  = 2;
-        assert_ok!( SUT::set(admin, resource_id, local_addr) );
+        let admin = Origin::root();
+        let resource_id = [1; 32];
+        let local_addr  = [2; 32];
+        assert_ok!( BridgeMapping::set(admin, resource_id, local_addr) );
 
         // Check that resource mapping was added to storage
-        assert_eq!(SUT::addr_of(resource_id), Some(local_addr));
-        assert_eq!(SUT::name_of(local_addr), Some(resource_id));
+        assert_eq!(BridgeMapping::addr_of(resource_id), Some(local_addr));
+        assert_eq!(BridgeMapping::name_of(local_addr), Some(resource_id));
     });
 }
 
 #[test]
 fn set_resource_updates_existing_storage() {
     new_test_ext().execute_with(|| {
-        let admin       = Origin::root();
-        assert_ok!( SUT::set(admin.clone(), 1, 2) );
-        assert_ok!( SUT::set(admin        , 1, 3) );
+        let admin = Origin::root();
+        let resource_id = [1; 32];
+        let local_addr  = [2; 32];
+        assert_ok!( BridgeMapping::set(admin.clone(), resource_id, local_addr));
+
+        let resource_id = [1; 32];
+        let local_addr  = [3; 32];
+        assert_ok!( BridgeMapping::set(admin, resource_id, local_addr));
 
         // Check that resource mapping was added to storage
-        assert_eq!(SUT::addr_of(1), Some(3));
-        assert_eq!(SUT::name_of(3), Some(1));
+        assert_eq!(BridgeMapping::addr_of(resource_id), Some(local_addr));
+        assert_eq!(BridgeMapping::name_of(local_addr), Some(resource_id));
     });
 }
 
 #[test]
 fn non_admin_cannot_set_resource() {
     new_test_ext().execute_with(|| {
-        let user        = Origin::signed(0);
-        let resource_id = 1;
-        let local_addr  = 2;
-        assert_err!(SUT::set(user, resource_id, local_addr),
-                    sp_runtime::traits::BadOrigin);
+        let user = Origin::signed(0);
+        let resource_id = [1; 32];
+        let local_addr  = [2; 32];
+        assert_err!(BridgeMapping::set(user, resource_id, local_addr), sp_runtime::traits::BadOrigin);
```

Comment:
Can we maybe use `assert_nop` instead of the `assert_err` to ensure, that we don't modify the state?